django/views/csrf.py

```diff
@@ -131,4 +130,8 @@ def csrf_failure(request, reason=""):
         'docs_version': get_docs_version(),
         'more': _("More information is available with DEBUG=True."),
     })
+    try:
+        t = loader.get_template(template_name)
+    except TemplateDoesNotExist:
+        t = Engine().from_string(CSRF_FAILURE_TEMPLATE)
```

Comment:
I agree that always attempting to load a `'csrf_failure.html'` is a good idea.

The issue I'm pointing at is the following. If a user was to specify a custom template name with a typo I think a `TemplateDoesNotExist` error should be raised instead of simply silencing it and falling back to the `CSRF_FAILURE_TEMPLATE` string.